book/tools.md

```diff
@@ -93,11 +97,32 @@ Automated code format repair.
 
 * [Apache Airflow](https://airflow.apache.org/)
 * [targets R package](https://wlandau.github.io/targets-manual/)
+* [Hamilton (Python)](https://hamilton-docs.gitbook.io/docs/)
+
+## Dependency management
+
+* [renv (R)](https://rstudio.github.io/renv/articles/renv.html)
 
 ## Continuous Integration Platforms
 
 * [GitHub Actions](https://github.com/features/actions)
 * [GitLab CI/CD](https://docs.gitlab.com/ee/ci/)
 * [Travis](https://travis-ci.org/)
 * [Jenkins](https://www.jenkins.io/)
-* [Coveralls](https://coveralls.io/)
\ No newline at end of file
+* [Coveralls](https://coveralls.io/)
+
+## Data processing, analysis, and publishing
+
+### Data linkage
+
+* [Splink (Python)](https://www.gov.uk/government/publications/joined-up-data-in-government-the-future-of-data-linking-methods/splink-mojs-open-source-library-for-probabilistic-record-linkage-at-scale#introduction)
+
+### Data wrangling
```

Comment:
Could we swap this too:?
```suggestion
### Data engineering
```
I've been put off the term, because when people do data wrangling they're usually doing data engineering just without any quality assurance 😆